### ISSUE TITLE
StockholmSideBar Exception when scrolling

### ISSUE THREAD

**Issue by getBoolean** [serverpod/stockholm#1]
When scrolling StockholmSideBar, it throws an exception. This can be seen in the example app if you make the screen small enough.

Output:
```
Exception has occurred.
FlutterError (The provided ScrollController is currently attached to more than one ScrollPosition.
The Scrollbar requires a single ScrollPosition in order to be painted.
When the scrollbar is interactive, the associated Scrollable widgets must have unique ScrollControllers. The provided ScrollController must be unique to a Scrollable widget.)
```

**Comment by getBoolean** [comment]
Have you been able to take a look at this?

**Comment by getBoolean** [comment]
Fixed in Flutter 3.3 due to the default behavior changing for desktop apps
